src/VisualStudio/Core/Def/Implementation/ProjectSystem/Legacy/AbstractLegacyProject.cs

```diff
@@ -109,7 +107,6 @@ internal abstract partial class AbstractLegacyProject : ForegroundThreadAffiniti
             try
             {
                 _externalErrorReporter = new ProjectExternalErrorReporter(VisualStudioProject.Id, externalErrorReportingPrefix, serviceProvider);
-                _editAndContinueProject = new VsENCRebuildableProjectImpl(Workspace, VisualStudioProject, serviceProvider);
             }
             catch (Exception)
```

Comment:
💡  Now this can only fail if `serviceProvider` is null or if it fails to provide an instance of `VisualStudioWorkspace`. This won't happen in production, so if tests still pass with this removed it would be safe to remove the catch-all. Most of the situations that could lead to this were inside the `VsENCRebuildableProjectImpl` constructor.

Reply:
Removed.